bigquery2/Module.md

```diff
@@ -60,19 +60,64 @@ bigquery2:Client bigqueryClient = new(bigqueryConfig);
 ```
 
 
-The `listProjects` function lists all projects to which current user have been granted any project role.
+The `listProjects` remote function lists all projects to which current user have been granted any project role.
 ```ballerina
 // List projects.
-var projects = bigqueryClient->listProjects();
+var listProjectResponse = bigqueryClient->listProjects();
 ```
 
 The response from `listProjects` is a `ProjectList` object if the request was successful or an `error` on failure.
 
 ```ballerina
-    if (projects is bigquery2:ProjectList) {
-            io:print("Projects: ", projects);
-        } else {
-            io:println("Error: ", response);
-        }
+    if (listProjectResponse is bigquery2:ProjectList) {
+        io:print("Projects: ", listProjectResponse);
+    } else {
+        // Error will be printed.
+        io:println("Error: ", listProjectResponse);
     }
 ```
+
+
+#### Sample
+```ballerina
+import wso2/bigquery2;
+
+import ballerina/config;
+import ballerina/http;
+import ballerina/io;
+
+// Create the Bigquery configuration.
+bigquery2:BigqueryConfiguration bigqueryConfig = {
+    clientConfig: {
+        auth: {
+            scheme: http:OAUTH2,
+            accessToken: config:getAsString("ACCESS_TOKEN"),
+            clientId: config:getAsString("CLIENT_ID"),
+            clientSecret: config:getAsString("CLIENT_SECRET"),
+            refreshToken: config:getAsString("REFRESH_TOKEN")
+        }
+    }
+};
+
+// Create the Bigquery client.
+bigquery2:Client bigqueryClient = new(bigqueryConfig);
+
+public function main() {
+
+    // Invoke listProjects function to list the projects.
```

Comment:
```suggestion
    // Invoke the `.listProjects()` remote function to list the projects.
```

Let's fix everywhere.

Reply:
Isn't it better to say `listProjects` because the function name don't need `.` and we specifically say `remote function`? If we didn't specifically say the word `function`, it makes sense to add `()` to the name to imply it is a function. WDYT?

E.g.-
// Invoke the `listProjects` remote function to list the projects.

Reply:
`listProjects()`